### ISSUE TITLE
is not working in ubuntu

### ISSUE THREAD

**Issue by vpsone** [microsoft/inshellisense#254]
**Describe the bug**
is not working. whenever I type `is` command it goes to live but it does not show auto-completion

**Expected behavior**
It should work correctly.

**Environment**
 - OS: Ubuntu 22.04
 - Output of `is --version`: 0.0.1-rc.15
 - Nodejs Version:  v18.14.0
 - Shell:  zsh
 - Shell Version: 5.8.1

**Additional context**
Here is my log file
```log
{"msg":"load local spec failed","e":"Cannot find module '/home/vps/.fig/autocomplete/build/index.js' imported from /home/vps/.nvm/versions/node/v18.14.0/lib/node_modules/@microsoft/inshellisense/build/runtime/runtime.js","specPath":"/home/vps/.fig/autocomplete/build"}
{"msg":"writing data","data":"\u001b[2J\u001b[3J\u001b[H"}
{"msg":"parsing data","data":"\u001b[1m\u001b[7m%\u001b[27m\u001b[1m\u001b[0m \r \r","bytes":{"0":27,"1":91,"2":49,"3":109,"4":27,"5":91,"6":55,"7":109,"8":37,"9":27,"10":91,"11":50,"12":55,"13":109,"14":27,"15":91,"16":49,"17":109,"18":27,"19":91,"20":48,"21":109,"22":32,"23":32,"24":32,"25":32,"26":32,"27":32,"28":32,"29":32,"30":32,"31":32,"32":32,"33":32,"34":32,"35":32,"36":32,"37":32,"38":32,"39":32,"40":32,"41":32,"42":32,"43":32,"44":32,"45":32,"46":32,"47":32,"48":32,"49":32,"50":32,"51":32,"52":32,"53":32,"54":32,"55":32,"56":32,"57":32,"58":32,"59":32,"60":32,"61":32,"62":32,"63":32,"64":32,"65":32,"66":32,"67":32,"68":32,"69":32,"70":32,"71":32,"72":32,"73":32,"74":32,"75":32,"76":32,"77":32,"78":32,"79":32,"80":32,"81":32,"82":32,"83":32,"84":32,"85":32,"86":32,"87":32,"88":32,"89":32,"90":32,"91":32,"92":32,"93":32,"94":32,"95":32,"96":32,"97":32,"98":32,"99":32,"100":32,"101":32,"102":32,"103":32,"104":32,"105":32,"106":32,"107":32,"108":32,"109":32,"110":32,"111":32,"112":32,"113":32,"114":32,"115":32,"116":32,"117":32,"118":32,"119":32,"120":32,"121":32,"122":32,"123":32,"124":32,"125":32,"126":32,"127":32,"128":32,"129":32,"130":32,"131":32,"132":32,"133":32,"134":32,"135":32,"136":32,"137":32,"138":32,"139":32,"140":32,"141":32,"142":13,"143":32,"144":13}}
{"msg":"writing data","data":"\u001b[1m\u001b[7m%\u001b[27m\u001b[1m\u001b[0m  \r \r"}
{"msg":"parsing data","data":"\r\u001b[0m\u001b[27m\u001b[24m\u001b[J\r\n\u001b[1;36m~\u001b[0m \r\n\u001b[1;32m❯\u001b[0m \u001b[K","bytes":{"0":13,"1":27,"2":91,"3":48,"4":109,"5":27,"6":91,"7":50,"8":55,"9":109,"10":27,"11":91,"12":50,"13":52,"14":109,"15":27,"16":91,"17":74,"18":13,"19":10,"20":27,"21":91,"22":49,"23":59,"24":51,"25":54,"26":109,"27":126,"28":27,"29":91,"30":48,"31":109,"32":32,"33":13,"34":10,"35":27,"36":91,"37":49,"38":59,"39":51,"40":50,"41":109,"42":111,"43":27,"44":91,"45":48,"46":109,"47":32,"48":27,"49":91,"50":75}}
```

**Comment by ditsuke** [comment]
Can reproduce with `zsh`. Works on `bash`

**Comment by cpendery** [comment]
This should be fixed by #275 & #276 (probably should have titled them fixes for *nix, not macOS) . I'm now able to run inshellisense on ubuntu w/ zsh. Please look for release `0.0.1-rc.16` for the fixes
